files/build_templates/lldp.timer.j2

```diff
@@ -1 +0,0 @@
-per_namespace/lldp.timer.j2
```

Comment:
Does the actual file need to be delelted as well, not just the symlink?

Reply:
Actual file doesn't exist

Reply:
The file is here: https://github.com/sonic-net/sonic-buildimage/blob/master/files/build_templates/per_namespace/lldp.service.j2

Removing the symbolic link might break chassis? @arlakshm

Reply:
That's the `lldp.service` file, not the `lldp.timer` file. It looks like the timer was removed in #13969.